modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/AgentsService.java

```diff
@@ -265,16 +266,27 @@ public void revokeToken(Account acc, String oldTok) {
      * @param req Request.
      * @throws IllegalStateException If connected agent not founded.
      */
-    void sendLocally(AgentRequest req) throws IllegalStateException, IOException {
+    void sendLocally(AgentRequest req) throws IllegalStateException, IllegalArgumentException, IOException {
```

Comment:
Please update `@throws` section in javadocs.

Reply:
Updated